bentoml/cli/deployment.py

```diff
@@ -95,9 +97,17 @@ def _format_labels_for_print(labels):
     return '\n'.join(result)
 
 
+def _format_deployment_age_for_print(deployment_pb):
+    if not deployment_pb.created_at:
+        return None
```

Comment:
add a comment about deployments created before 0.4.5 does not have this field in database